package.json

```diff
@@ -30,6 +30,7 @@
     "dist"
   ],
   "dependencies": {
+    "@relaypro/sdk": "^2.4.0",
```

Comment:
Remove `@relaypro/sdk`... this is actually referencing itself. :)

And reset package-lock.json changes